### PR TITLE
Add guava dependency to the "input" project

### PR DESCRIPTION
**Pull request by serhii-kozionov**
Fixes an issue with a missing guava dependency after #59. 
Also bumps the guava version to `29.0`.

### CODE REVIEW

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov Maybe add guava to the whole `JxBrowser-Examples` project? For now, only 2 submodules uses it.

File: input/build.gradle [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ *  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ *  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ *  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+dependencies {
+    // Depend on Guava for the Uninterruptibles utility class to wait for the latch.
```
Comment (vladimir-ikryanov) [inline input/build.gradle:22]: I don't think we need to explain why we add some dependency.

**Review by vladimir-ikryanov — CHANGES_REQUESTED** [review]

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov PTAL

**Review by vladimir-ikryanov — APPROVED** [review]
LGTM

**Pull request merged**